docs/Making-a-release.md

```diff
@@ -0,0 +1,11 @@
+# Making a Release of Simulation Construction Set 2
+1. Bump the version in `group.gradle.properties`
+2. Commit only `group.gradle.properties` with a message in the format: "`:bookmark: <version>`"
+3. Create a tag with the version name
+4. Push the release commit and tag
+5. Ensure `publishUsername` and `publishPassword` are set in `~/.gradle/gradle.properties`
```

Comment:
Set to what? Github account? Something special?

Reply:
This would be the credentials for the maven repository. In the case of SCS2, it's sonatype maven central.